canvas_modules/common-canvas/__tests__/common-properties/actions/button-action-test.js

```diff
@@ -17,19 +17,23 @@
 import React from "react";
 import { Provider } from "react-redux";
 import ActionButton from "./../../../src/common-properties/actions/button";
-import { render } from "../../_utils_/mount-utils.js";
 import { expect } from "chai";
-import { expect as expectJest } from "@jest/globals";
+import { afterEach, beforeEach, expect as expectJest } from "@jest/globals";
```

Comment:
Why did you need to add `afterEach` and `beforeEach`?  These should already be globally defined.